modules/ROOT/pages/6.0-release-notes-known-issues.adoc

```diff
@@ -23,4 +23,13 @@ As of this release, however, the specific cause is still being investigated.
 Workaround::
 Other than keeping `nonbreaking_force_tab` set to `false`, no workaround is currently available.
 
-// end::known-issues[]
\ No newline at end of file
+== Editor content can overlap with an open floating toolbar drawer when using certain premium skins
+
+In {productname} 6.0, if one of the premium skins, `snow`, `naked`, `outside`, or `fabric` is used and the floating toolbar drawer is open, content within the editor may visually overlap the toolbar drawer.
+
+The expected behavior is for there to be no overlap between the editor content and the toolbar drawer.
+
+Workaround::
+There was no known workaround at the time of the release.
```

Comment:
Wouldn't the workaround here be to use a different skin? I seem to recall that being mentioned at sprint open at least.

Reply:
Actually... a better workaround would be to use a toolbar mode other than `floating`, such as `sliding`.

Reply:
Maybe something like this?

```suggestion
Use a toolbar mode other than the default `floating` mode. For details, see the xref:toolbar-configuration-options.adoc#toolbar_mode[`+toolbar mode+` option]. Otherwise, use a different premium skin than noted above. For details, see: xref:premium-skins-and-icons.adoc[Tiny Skins and Icon Packs].
```

Reply:
LGTM 👍